runmariadb.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/bash
+PATH=$PATH:/usr/sbin/
+DATADIR="/var/lib/mysql"
+MARIADB_PASSWORD=${MARIADB_PASSWORD:-"change_me"}
+
+if [ ! -d "$DATADIR/mysql" ]; then
+    crudini --set /etc/my.conf mysqld max_connects 64
+    crudini --set /etc/my.conf mysqld max_heap_table_size 1M
+    crudini --set /etc/my.conf mysqld innodb_buffer_pool_size 5M
+    crudini --set /etc/my.conf mysqld innodb_log_buffer_size 512K
+
+    mysql_install_db --datadir="$DATADIR"
+
+    chown -R mysql /var/log/mariadb
+    chown -R mysql "$DATADIR"
+
+    cat > /tmp/configure-mysql.sql <<-EOSQL
+        DELETE FROM mysql.user ;
```

Comment:
I think the last change may be causing a problem. When I test this now the container exits and I see:
$ sudo podman logs mariadb
/bin/runmariadb: line 30: warning: here-document at line 17 delimited by end-of-file (wanted `EOSQL')
/bin/runmariadb: line 31: syntax error: unexpected end of file

I think the indenting for these lines has to go back to the first column, the last change introduced the intending.

Reply:
you're right, I don't know why this changed got actually reverted during my last commit
it should be back to normal now